800.renames-and-merges/ruby.yaml

```diff
@@ -117,6 +117,7 @@
 - { setname: "ruby:bundler",           name: bundler }
 - { setname: "ruby:bundler",           namepat: "ruby[0-9.]+-bundler" }
 - { setname: "ruby:gems",              name: ["ruby:rubygems",ruby-gems,rubygems] }
+- { setname: "ruby:goocanvas",         namepat: "ruby-goocanvas[0-9.]+" }
```

Comment:
Since the package was matched by wildcard rule for ruby packages before, it'll be named `ruby:goocanvasXXX` when this rule is checked.